src/replay/execution.zig

```diff
@@ -696,30 +696,26 @@ pub const BlockError = enum {
 
 pub const LogHelper = struct {
     logger: Logger,
-    last_active_slots: ?[]const Slot,
+    // we store a hash of the previous set of active slots, to avoid printing duplicate sets
+    last_active_slots_hash: ?Hash,
     slots_are_the_same: bool,
 
     pub fn init(logger: Logger) LogHelper {
         return .{
             .logger = logger,
-            .last_active_slots = null,
+            .last_active_slots_hash = null,
             .slots_are_the_same = false,
         };
     }
 
-    /// takes ownership of active_slots,
-    pub fn logActiveSlots(
-        self: *LogHelper,
-        active_slots: []const u64,
-        deinit_allocator: Allocator,
-    ) void {
-        self.slots_are_the_same = if (self.last_active_slots) |last_slots|
-            std.mem.eql(u64, active_slots, last_slots)
+    pub fn logActiveSlots(self: *LogHelper, active_slots: []const u64) void {
+        const active_slots_hash = Hash.generateSha256(std.mem.sliceAsBytes(active_slots));
```

Comment:
I'm happy to merge this as-is but a thought just occurred to me. You could use FnvHasher here instead, which would be more efficient. It doesn't really matter though since this function is not called very often. Feel free to resolve this conversation.

```suggestion
        const active_slots_hash = FnvHasher.hash(std.mem.sliceAsBytes(active_slots));
```

Reply:
I think you're right, yeah, sha256 is overkill. Going to resolve to get this in asap though!